src/swerex/server.py

```diff
@@ -30,6 +30,10 @@
 app = FastAPI()
 runtime = LocalRuntime()
 
+_last_processed_request_id = None
+_last_processed_response = None
```

Comment:
nit: Since the other variables here are not prefixed with `_`, i'd just name these `last_processed_request_id` and `last_processed_response` to match the surrounding code